pkg/command/command_test.go

```diff
@@ -74,6 +75,30 @@ func TestExecutableRun(t *testing.T) {
 	})
 }
 
+func TestEnvironmentVariables(t *testing.T) {
+
+	ExecCommand = helperCommand
+	defer func() { ExecCommand = exec.Command }()
+
+	o := new(bytes.Buffer)
+	e := new(bytes.Buffer)
```

Comment:
I would agree with Florian to use more speaking names here.

Reply:
done.